README.md

```diff
@@ -173,6 +173,7 @@ Options
 	--tsconfig       Specify the path to a custom tsconfig.json
 	--css-modules    Configures .css to be treated as modules (default: null)
 	-h, --help       Displays this message
+	--closure				 Specify if using the closure-compiler, when false microbundle will use terser instead (default: true)
```

Comment:
If the default is to use closure, the option should be `--terser`. See other comment for discussion of whether default is correct here.